numpy/core/src/umath/ufunc_type_resolution.c

```diff
@@ -580,18 +576,17 @@ PyUFunc_SimpleUniformOperationTypeResolver(
 NPY_NO_EXPORT int
 PyUFunc_AbsoluteTypeResolver(PyUFuncObject *ufunc,
                                 NPY_CASTING casting,
-                                PyArrayObject **operands,
-                                PyObject *type_tup,
+                                PyArray_Descr **op_dtypes,
                                 PyArray_Descr **out_dtypes)
 {
     /* Use the default for complex types, to find the loop producing float */
-    if (PyTypeNum_ISCOMPLEX(PyArray_DESCR(operands[0])->type_num)) {
-        return PyUFunc_DefaultTypeResolver(ufunc, casting, operands,
-                    type_tup, out_dtypes);
```

Comment:
From the style guide, I'd be inclined to leave the indentation as it was:

> * Breaking long lines: if you can, break after commas in the
>   outermost argument list.  Always indent continuation lines
>   appropriately, e.g., ::
> 
>         PyErr_SetString(PyExc_TypeError,
>                 "Oh dear, you messed up.");
> 
>   Here appropriately means at least two tabs. It isn't necessary to
>   line everything up with the opening parenthesis of the function
>   call.

In particular, I've had cases in the past in numpy where I've bulk renamed an internal function - if you align things this way, then you force the person doing that type of change to manually find all the wrapped whitespace lines too.

Reply:
Yeah. The exception is a nice example where it makes a lot sense. I do no care much either way, and if I just change dthe style for no reason, shouldn't have done that.